index.html

```diff
@@ -73,18 +71,7 @@ <h1 id="title">freeCodeCamp Survey Form</h1>
 
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
 
-    <!--
-
-    Hello Camper!
-
-    For now, the test suite only works in Chrome!
-    Please read the README below in the JS Editor before beginning.
-    Feel free to delete this message once you have read it.
-    Good luck and Happy Coding!
-
-    - The freeCodeCamp Team
-
-    -->
     <script  src="js/index.js"></script>
+    </section>
     </body>
 </html>
```

Comment:
- [ ] <b>REQUIRED:</b> This is an example as of how to use inline comments for required change.